### DISCUSSION TITLE
Project dead?

### DISCUSSION THREAD

**Discussion by Perkolator** [Cookie-AutoDelete/Cookie-AutoDelete#1637] [Q&A / Support]
There haven't been any visible development for over a year now. Is this project dead?

**Comment by jkuj6** [comment]
Id love to know the same thing.. i was about to open an issue on this. I hope some contributor or the maintainer can announce something, because i rely on this addon a lot.

**Comment by kennethtran93** [comment]
Maybe it was, but I do want to put some more time into this project but alas I don't have much of it left.  Project is open for Pull Requests as well if it helps to enhance/fix stuff.  Looking to see what I can do to make it at least manifest v3 to start.

**Reply by pstemari** [reply]
If manifest v3 support is feasible, that would be fantastic. At the moment Google is silently disabling the extension.

I'd offer to help, but I break out in a rash every time I touch JavaScript :-)

**Reply by conana** [reply]
@kennethtran93 what help do you need? 

Off the top of my head, I imagine the primary blocker of help is around coordination and knowledge. e.g.

- is CAD functionality in theory supported by V3 or crippled by it?
- are you working currently on Manifest V3 support so no help needed or is there useful work folk could contribute?
- if there's useful work to be done, do you already have the clarity on it and the time to manage a task list, put out a call to action and see if there are any takers?

**Comment by Simlock** [comment]
A really good extension. Unfortunately, I can't see any activity for further development. It really looks like the project is not being continued.

**Comment by Quinkink** [comment]
Chrome has asked to disable this extension on the browser. I've overidden it for now. With a score of 4.5/5 stars it would be a darn shame for this to extension to die.

**Comment by bandaid** [comment]
There are flags you can use to bypass the updates which disable v2 manifest. 

https://www.reddit.com/r/uBlockOrigin/comments/1itw1bz/end_of_support_for_ubo_on_chrome_chromium/

In Chrome 140, apparently, these will be gone. So you'll need to find a different browser and slowly take yourself off Chrome. Vivaldi is a great browser alternative.

**Reply by Perkolator** [reply]
> Vivaldi is a great browser alternative.

You do realize that Vivaldi is based on Chromium? Thus all v2 manifest extensions WILL stop working in it.. and Vivaldi team hasn't promised to keep supporting v2, unlike some other Chromium based browser vendors.

**Comment by colopatiron** [comment]
It wouldn't be so bad if this project dropped support for Chrome altogether.

**Reply by kelghu** [reply]
Why? This is by far the best cookie autodelete extension out there

**Comment by JulianAtkins** [comment]
Someone has already forked this project and implemented Manifest V3 !!! Also made a new release:

https://github.com/median-dxz/Cookie-AutoDelete-MV3/releases

**Reply by Perkolator** [reply]
From that repo:

> ## About This Fork
> 
>The main purpose of this fork is to **help migrate the original repository to Manifest V3 (Google Manifest v3) and adopt a more modern JavaScript build toolchain**. As the creator of this fork, I have limited time available, so I won't be spending much time on this fork, nor will I be uploading its build to the Chrome Web Store. Therefore, until the primary goal is achieved, this project will not accept requests for new features.
> 
> **Please note that you need to load the build from Release into your browser via developer mode.**
